guide/english/book-recommendations/index.md

```diff
@@ -249,3 +249,26 @@ https://books.goalkicker.com/OracleDatabaseBook/
 This list was compiled from multiple suggestion threads on Reddit and Stackoverflow. 
 
 Please feel free to add more that you have found useful!
+
+My Changes
```

Comment:
Please delete "My Changes" line.

Reply:
You want me to delete what I just added?

Reply:
Just the "My Changes" line. You can directly add your changes without adding "My Changes".

Reply:
Okay. Thank you.

Reply:
I don't understand what this means  #26726 failed
I also don't understand what 'Outdated' mean.
Is something wrong with my addition, and what should I do?

Reply:
@Aynanufi You were able to merge in your own PR successfully. #26726 seems to be someone else's Pull Request.

Also, you needn't worry about 'Outdated'. It just indicates new changes have been pushed since the last review.

However, you added a space between the `[]` and `()` in the links your most recent changes: 

> [Amazon] (https://www.amazon.com/Python-Kids-Playful-Introduction-Programming-ebook/dp/B00ADX21Z6)

If you remove the space, the text "Amazon" becomes a clickable link, which is how all links in the guide articles are formatted, like so -

> [Amazon](https://www.amazon.com/Python-Kids-Playful-Introduction-Programming-ebook/dp/B00ADX21Z6)

Please make this change for all your links, and it should be good to go.